multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -163,6 +142,7 @@ func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctr
 		klog.ErrorS(err, "Failed to add member cluster to ClusterSet", "cluster", memberAnnounce.ClusterID)
```

Comment:
We should ignore "already exists" error in addMemberToClusterSet (in my mind better to add comments on addMemberToClusterSet to explain the behavior)?

Reply:
Yes, return nil for `already exists` error.

Reply:
Oh, I did not see that. Then we should move r.addOrUpdateMemberStatus() to line 113 before "if common.StringExistsInSlice(memberAnnounce.Finalizers, finalizer)"?

I assume we can call the func only once, and remove the call at line 95.